src/chrome-launcher.ts

```diff
@@ -205,7 +209,7 @@ class Launcher {
       }
     }
     if (this.chromePath === undefined) {
-      const installations = await chromeFinder[getPlatform() as SupportedPlatforms]();
+      const installations = (<typeof Launcher>this.constructor).getInstallations();
```

Comment:
can't we just call `Launcher.getInstallations()`? :)

Reply:
wow you were fast! thanks for hopping in so quickly!

I was proactively typing up a note for why I opted for this but was obvs too slow lol --I'll respond back in the other thread.

Reply:
addressed in other thread --closing out :)